### ISSUE TITLE
replace RelatedPerson with Person for SMART server identifying user

### ISSUE THREAD

**Issue by isaacvetter** [smart-on-fhir/smart-on-fhir.github.io#148]
@whitehatguy laid out these rules on the [SMART google group](https://groups.google.com/forum/#!msg/smart-on-fhir/1F0jcaKxGsg/3Ua3bRgqDAAJ) to define what FHIR resource should be used to represent the current user with the profile/fhirUser claim:

1. For practitioners, use "Practitioner" resource.
2. If the records being shared are those of the authenticated user, use "Patient" resource.
3. For others, use "Person" resource.

We should replace references to RelatedPerson with Person.

**Comment by jmandel** [comment]
We should discuss the merits of saying "Always use `Person` when it's not a Patient or Practitioner" vs saying "Use `RelatedPerson` if possible and `Person` if not". The key point is that RelatedPerson captures the specific details/relation, and superficially at least it makes sense to use/keep this when possible (rather than prohibiting it).

**Comment by isaacvetter** [comment]
Hey @jmandel ,

Why not always used RelatedPerson, but sometimes have an empty RelatedPerson.relationship ?

**Comment by michelemottini** [comment]
In our server we have pretty much to use Person: you log-in as a user - that does not really have any corresponding FHIR resource beside person,. and a user can be a patient or a practitioner or neither. 

We have also RelatedPerson in our database, but as a completely separate concept from users - so not really usable in connection with the authorization process

**Comment by isaacvetter** [comment]
Hey @michelemottini - Practitioner essentially maps to "user" in non-patient facing apps. The Person FHIR resource seems to be intentionally abstract and primarily intended for complex identity mapping. 

In your system, do your Person resources contain a link reference to Practitioner, Patient or RelatedPerson?

**Comment by michelemottini** [comment]
I our system Person (user) can link to a Practitioner or to a Patient or neither. Never to a RelatedPerson

**Comment by daliboz** [comment]
RelatedPerson (as a singular) doesn't work if I'm logging is as mom with access to my dependents or spouses. Essentially, any "patient" login where I have user/* scopes.

**Comment by jmandel** [comment]
Agreed @daliboz! And thanks for the context @michelemottini. I think the decision we'll want to make is: which subset of {Patient, Practitioner, RelatedPerson, Person} we want to support at the level of the specification.

We've heard use cases for all four, on my view  -- though that doesn't mean we need to support all four.
